src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.CompilationTracker_Generators.cs

```diff
@@ -121,6 +121,7 @@ private partial class CompilationTracker : ICompilationTracker
             var projectId = this.ProjectState.Id;
             var infosOpt = await connection.TryInvokeAsync(
                 compilationState,
+                projectId,
                 (service, solutionChecksum, cancellationToken) => service.GetSourceGenerationInfoAsync(solutionChecksum, projectId, cancellationToken),
```

Comment:
>(service, solutionChecksum, cancellationToken) [](http://example.com/codeflow?start=16&length=46)

Would it make sense for this callback to include the projectId?

Reply:
will look into that :)